.github/workflows/performance-test.yml

```
@@ -0,0 +1,169 @@
+          echo "⚠️ 성능 테스트 결과 파일을 찾을 수 없습니다."
+        fi
+        
+        echo ""
+        echo "=== 테스트 환경 정보 ==="
+        echo "- 동시 사용자 수: ${{ github.event.inputs.concurrent_users }}"
+        echo "- 테스트 시간: ${{ github.event.inputs.test_duration }}초"
+        echo "- 서버 환경: GitHub Actions (ubuntu-latest)"
+        echo "- 테스트 대상: Mock FastAPI 서버"
+        
+    - name: 결과 파일 업로드
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)